docs/testing.md

```diff
@@ -0,0 +1,140 @@
+## Testing
+
+`tsaga` comes with its testing utilities.
+
+The utilities are similar to `redux-saga-test-plan` and modeled after how a saga would run in your application.
+
+### Sagas
+
+The assertions use the `expect` function from the `jest` testing framework.
+
+```typescript
+import { expectSaga, select, call, run, spawn } from 'tsaga';
+
+it('should run a saga', () => {
+  return (
+    expectSaga(watchForFetchUser)
+      // Setup a store with a reducer and an optional initial state
+      // This is optional
+      .withReducer(rootReducer, undefined)
+      // Setup some mocks which should be used instead of the real functions
+      // The mock helper functions provide useful type assertions,
+      // so that you return a value which is valid
+      // This is optional
+      .withMocks([
+        // A select mock, the second argument is the return value when this mock is used
```

Comment:
This is true, but probably also something we really should fix (and support select and call parameters for mocks)

Reply:
https://github.com/tp/tsaga/issues/34